### PR TITLE
add merlin mainnet& merlin testnet

### PR DESCRIPTION
**Pull request by npc7**
Merlin Mainnet RPC:rpc.merlinchain.io
Merlin Testnet RPC:test-rpc.merlinchain.io

### CODE REVIEW

**Review by nlordell — CHANGES_REQUESTED** [review]
This PR has a lot of unrelated changes (in particular, it adds the `dist` folder).

Please remove the `dist` folder (and restore it in the `.gitignore`).

**Comment by npc7** [comment]
sorry ,my fault， repush finished ，thx

File: .gitignore [hunk 1]
```diff
@@ -6,7 +6,7 @@ node_modules/
 deployments/
 env/
 .env
-dist/
+/dist
```
Comment (nlordell) [inline .gitignore:9]: Can you undo this change?
```suggestion
dist/
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@safe-global/safe-deployments",
-  "version": "1.35.0",
+  "version": "1.35.1",
```
Comment (nlordell) [inline package.json:3]: And this one as well, as we have separate processes for managing the package version and release cycle:

```suggestion
  "version": "1.35.0",
```

Comment (npc7) [reply package.json]: done

File: .gitignore (hunk 1 above)
Comment (nlordell) [reply .gitignore:9]: Can you undo this change as well?

**Comment by mmv08** [comment]
Closing due to inactivity. Please feel free to open a new PR.
